packages/mds-db/tests/mds-db.spec.ts

```diff
@@ -437,9 +437,9 @@ if (pg_info.database) {
 
       it('understands the summary parameter', async () => {
         const geographiesWithoutGeoJSON = await MDSDBPostgres.readGeographies({ summary: false })
-        geographiesWithoutGeoJSON.forEach(geography => assert(geography.geography_json))
+        geographiesWithoutGeoJSON.forEach((geography: Geography) => assert(geography.geography_json))
```

Comment:
Why is the `geography` explicitly typed? That implies to me that the return type of `readGeographies` is incorrect.

Reply:
I guess because the `geojson` might not be there when `summary` is true, but I'll set it to null on each geography returned in `readGeographies`.